packages/commerce-sdk-react/README.md

```diff
@@ -64,7 +95,224 @@ const App = ({children}) => {
 
 export default App
 ```
-### Useful Links:
+
+## Shopper Authentication and Token Management
+
+_💡 This section assumes you have read and completed the [Authorization for Shopper APIs](https://developer.salesforce.com/docs/commerce/commerce-api/guide/authorization-for-shopper-apis.html) guide._
+
+To help reduce boilerplate code for managing shopper authentication, by default, this library automatically initializes shopper session and manages the tokens for developers. Currently, the library supports the [Public Client login flow](https://developer.salesforce.com/docs/commerce/commerce-api/guide/slas-public-client.html).
+
+### Shopper Session Initialization
+
+On `CommerceApiProvider` mount, the provider initializes shopper session by initiating the SLAS __Public Client__ login flow. The tokens are stored/managed/refreshed by the library.
+
+### Authenticate request queue
+
+While the library is fetching/refreshing the access token, the network requests will queue up until there is a valid access token.
+
+### Login helpers
+
+To leverage the managed shopper authentication feature, use the `useAuthHelper` hook for shopper login.
+
+Example:
+
+```jsx
+import {AuthHelpers, useAuthHelper} from '@salesforce/commerce-sdk-react'
+
+const Example = () => {
+    const register = useAuthHelper(AuthHelpers.Register)
+    const login = useAuthHelper(AuthHelpers.LoginRegisteredUserB2C)
+    const logout = useAuthHelper(AuthHelpers.LogOut)
+
+    return <button onClick={() => {
+        login.mutate({username, password})
```

Comment:
NIT: you might want to replace the variables username and password and provide example data. This will make it more obvious that if someone wants to copy and paste the code they have to change/provide that data, instead of ensuring there is some variables with those names defined somewhere.

Reply:
also give you an opportunity to put you name on things ;)

```
 login.mutate({username: 'kevin', password: 'pa$$word'})
```